openpype/plugins/publish/extract_burnin.py

```diff
@@ -156,6 +155,9 @@ def main_process(self, instance):
             filled_anatomy = anatomy.format_all(burnin_data)
             burnin_data["anatomy"] = filled_anatomy.get_solved()
 
+            # Add context data burnin_data.
+            burnin_data["context"] = instance.context.data["burnin_context"]
```

Comment:
This will crash as `burnin_context` may not be available on context. Rather use `instance.context.data.get("burnin_context") or {}`